powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/controller/ActivationController.java

```diff
@@ -141,7 +141,9 @@ public ObjectResponse<ActivationRemoveResponse> removeActivation(
             logger.debug("Signature validation failed");
             throw new PowerAuthSignatureInvalidException();
         }
-        if (!"3.0".equals(apiAuthentication.getVersion()) && !"3.1".equals(apiAuthentication.getVersion())) {
+        if (!"3.0".equals(apiAuthentication.getVersion())
```

Comment:
Just a note: This requires a note in the compatibility table (the latest SDKs would require this server version).

Reply:
I will add a note into migration guide and we will update documentation wherever applicable.